daisy_workflows/image_import/import_image.sh

```diff
@@ -141,6 +141,21 @@ function serialOutputKeyValuePair() {
   echo "<serial-output key:'$1' value:'$2'>"
 }
 
+function diskChecksum() {
+  set +x
+  CHECK_DEVICE=sdc
+  BLOCK_COUNT=$(cat /sys/class/block/$CHECK_DEVICE/size)
```

Comment:
This duplicated code seems reasonable given that this is an experiment. Can you add a comment pointing to the duplicated version (so that we know to remove it, and so that we remember to keep them in sync if they change)

Reply:
done